### DISCUSSION TITLE
Add CONTROL supported feature to conversation agent

### DISCUSSION THREAD

**Discussion by balloob** [home-assistant/architecture#1102] [Entity Models]
# Context

It's currently not clear from the state machine if a conversation agent is able to control Home Assistant.

# Proposal

Add a new ~~`API_ACCESS`~~ `CONTROL` supported feature to conversation entity

- Implementation: https://github.com/home-assistant/core/pull/121036
- OpenAI change: https://github.com/home-assistant/core/compare/conversation-supported-feature...openai-feature-has-api

# Impact

UI elements can inform the user if the conversation agent can access HA or not:

- Assist dialog can tell user if they are talking to a conversation agent that is not able to control Home Assistant
- Conversation agent selector can be limited to agents that can control Home Assistant

# Note

We only have a built-in API that offers Assist access for LLMs to conversation agents. However, custom components can register their own APIs. There is no knowing if those APIs allow accessing Home Assistant. That's why the feature flag is limited to `CONTROL`, and we can only know when an agent cannot control Home Assistant. It's unable for us to know what of Home Assistant can be controlled.

**Comment by frenck** [comment]
Discussed, and adjusted and merged. 👍
